htdocs/compta/stats/index.php

```diff
@@ -461,7 +461,7 @@
 			//var_dump($annee.' '.$year_end.' '.$mois.' '.$month_end);
 			if ($annee < $year_end || ($annee == $year_end && $mois <= $month_end)) {
 				if ($annee_decalage > $minyear && $case <= $casenow) {
-					if (!empty($cum[$caseprev]) && !empty($cum[$case])) {
+					if (!empty($cum[$caseprev]) && !empty($cum[$case]) && $cum[$caseprev] != 0) {
```

Comment:
The test 
!empty($cum[$caseprev])
is not enough ?

Reply:
No it's not because 
if $cum[$caseprev] = "0.0" (case already encountered)  then !empty($cum[$caseprev]) == true
This is due to the "." in the string

if !empty($cum[$case]) == true then the division by zero is done
So the test 
$cum[$caseprev] != 0 is important